src/pages/dormitory/call-names/index.tsx

```diff
@@ -0,0 +1,197 @@
+    const callNameInfo = callNameInfos[characterKey]
+    const characterInfo = characterInfos[characterKey]
+
+    const outlineStyle: CSSProperties = {
+      outlineColor: characterInfo.color,
+    }
+
+    return (
+      <>
+        {characterKeys.map(_characterKey => {
+          const callName = callNameInfo[_characterKey]
+
+          return (
+            <td key={_characterKey}>
+              <div>
+                {(() => {
+                  if (characterKey === _characterKey) {
+                    return callNameInfo.me.map(part => (
+                      <p style={outlineStyle} key={part} className="me">
+                        {part}
+                      </p>
+                    ))
+                  }
+
+                  if (callName == undefined) {
+                    return (
+                      <p style={outlineStyle} className="unknown">
+                        ?
+                      </p>
+                    )
+                  }
+
+                  return callName.split("/").map(part => (
+                    <p style={outlineStyle} key={part}>
+                      {part}
+                    </p>
+                  ))
+                })()}
+              </div>
+            </td>
+          )
+        })}
+        <td className="you">
+          <div>
+            {callNameInfo.you.map(part => (
+              <p style={outlineStyle}>{part}</p>
+            ))}
+          </div>
+        </td>
+      </>
+    )
+  }
+
+  return (
+    <Page showingHeaderOnTop={true}>
+      <Seo
+        title={`キャラクターの呼称表 | ボイボ寮 | VOICEVOX`}
+        description={
+          "ボイボ寮のキャラクターの呼び方一覧表です。必ずしも遵守する必要はなく、自由に改変して頂いても問題ありません。"
+        }
+      />
+
+      <div className="call-names-wrapper">
+        <section>
+          <h1 className="title">キャラクターの呼称表</h1>
+          <p>
+            ボイボ寮のキャラクターの呼び方一覧表です。必ずしも遵守する必要はなく、自由に改変して頂いても問題ありません。
+          </p>
+        </section>
+        <div className="call-names">
+          <table border={1}>
+            <thead>
+              <tr>
+                <th className="origin">
+                  <p>誰が</p>
+                  <p>誰を</p>
+                  <div />
+                </th>
+
+                {characterKeys.map(characterKey => {
+                  const characterInfo = characterInfos[characterKey]
+                  return (
+                    <th key={characterKey}>
+                      <Link to={`/dormitory/${characterInfo.id}/`}>
+                        {getCharacterImage(characterKey)}
+                        <p
+                          style={{
+                            color: characterInfo.color,
+                          }}
+                        >
+                          {characterInfo.name}
+                        </p>
+                      </Link>
+                    </th>
+                  )
+                })}
+                <th className="you">
+                  <p>全員</p>
+                  <p>(二人称)</p>
```

Comment:
中身が全角だったら全角括弧、みたいなルールをなんとなく考えてたりします 🙇 
```suggestion
                  <p>（二人称）</p>
```